packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -1295,7 +1295,7 @@ class _CpuProfileTimelineTree {
   }
 }
 
-extension CpuSamplesExtension on vm_service.CpuSamples {
+extension on vm_service.CpuSamples {
```

Comment:
should we be having nameless extensions everywhere we declare extension methods?

Reply:
There is a proposal to report unnecessarily public extensions. https://github.com/dart-lang/linter/issues/4851

But it's not a strong recommendation; I can revert this if desired; I was just cleaning up.

Reply:
so removing the `CpuSamplesExtension` name makes these extension methods private? Would renaming to `_CpuSamplesExtension` also have made these methods private?

Fine to leave nameless. A lot of these extensions have redundant names just because I / we may have thought the names were required.

Reply:
Right, unnamed or `_` would make it private. Yeah there is a lot of surprise in that lint rule suggestion that extensions can be unnamed, haha.